docs/user/multi-user-dfuse.md

```diff
@@ -0,0 +1,114 @@
+containers to other users, in this case the user running dfuse is the same as the user owning the
+containers so no modification of ACLs is required.
+
+### Automatic setting of ACLs.
+
+For containers the 'r', 't' and 'w' permissions are required on the container for the user running
+dfuse.  When containers are created though the `daos` command using a path which resides in
+a multi-user dfuse mount then these ACLs are set automaticlaly by the `daos` command and this
+is reported to the user.  (TODO: Check this is reported).
+
+If a container is created using a path not backed by multi-user dfure, for example on a per-user
+dfuse instance or where a `path` option is not provided at all then in order for
+anyone to access the container then the user will need to add this permission themselves.  This
+includes the user that owns the pool, without the dfuse users being able to access the container
+it cannot server the contents, even to the container owner.  This applies equally well to containers
+where the Unified Namespace link is created after the container itself is created.
+
+TODO: Check this, create a container single-user, access it via multi-user and check for errors,
+can 'daos cont set-acl' work on a dfuse-path in the multi-user setup?
+
+## Controlling data sharing and access.
+
+DFuse itself does not handle permissions checks regardless of options but rather provides file
+and directory ownership and permissions bits information to the kernel which then checks if user
+is entitedled to perform the requested operation.  As such once a multi-user dfuse instance is
+running then it will serve to the kernel the contents of any containers that the dfuse instance
+itself can access, and the kernel will then use standard Unix file/directory permissions to decide
+if operations are permitted.  It is therefore possible for users be able to access POSIX data across
+containers seamlessly where permissions is granted only at the POSIX file/directory layer.
+
+Consider the following example, in this case dfuse is configured and run in the recommended manner,
+with two users Anthony and Berlinda who use it to share data, yet Berlinda does not have ACL
+permissions to read Anthonys data, only POSIX permissions.
+
+Example:
+
+### Setup user to serve dfuse and create containers, mount points etc.
+sudo -u dserve dmg --insecure pool create root_pool --size 1g
+sudo -u dserve daos cont create --type POSIX root_pool root_container
+sudo mkdir /crate
+sudo chown dserve.dserve /crate
+
+### Run dfuse, this should be done via systemd to be automatically mounted at boot time.
+sudo -u dserve dfuse --multi-user --foreground /crate root_pool root_container
+
+### Create a directory for anthony to own, and create a pool for him.
+sudo mkdir -m 0700 /crate/anthony
+sudo chown anthony.anthony /crate/anthony
+sudo dmg --insecure pool create -u anyhony -g anthony anthony_pool --size 1g
```

Comment:
please remove insecure mode. it should not be used in production, and it might confuse readers that this is required in this case.